app/scripts/components/common/blocks/scrollytelling/index.tsx

```diff
@@ -358,42 +358,6 @@ function Scrollytelling(props) {
   return (
     <ScrollyMapWrapper>
       <TheMap topOffset={topOffset}>
-        <Styles>
```

Comment:
I think I do not understand why two layers are repeating. Is this for a smooth transition?

Reply:
This shouldn't happen. When the `SimpleMap` was wrapped with `Styles` the layer code was copied and this was not deleted. Great catch!